### DISCUSSION TITLE
How do I add custom icon to a table row?

### DISCUSSION THREAD

**Discussion by AlexKotik** [h2oai/wave#2134] [Q&A] [answered]
Okey, so I wanted to display some country stats in a table, but it would be more beatiful to use countries flags instead of country names or codes. I can get some good svg flag icons off the internet (free to use or buy a pack), but what would be the best way to paste svg icon into a table rows? I tried loading all the svg code into the memory for each flag and paste it into a markdown-kind column, it sort off works, but this looks like a hack and there should be a better way to do it. Any suggestions?

PS: Sorry if this thing is covered somewhere, but I couldn't find it in the docs.

**Comment by mturoci** [comment, marked as answer]
Hi @AlexKotik.

> it sort off works, but this looks like a hack

Correct. Unfortunately, there is no better way of achieving this as of September 2023. Tracked in https://github.com/h2oai/wave/issues/580.

**Reply by AlexKotik** [reply]
Btw do you think try switching to lightwave, add those svg flags as a static directory to FastAPI server and then render them as images would be a better idea?

**Reply by mturoci** [reply]
You can achieve the same with Wave as well via public/private dir. See https://wave.h2o.ai/docs/configuration/.

Lightwave is more suitable if you want full control over backend, e.g. custom auth etc. This is a UI problem so it wouldn't help much.

**Reply by mturoci** [reply]
Just [updated](https://github.com/h2oai/wave/commit/330908bb492c69634f78ede224596f865fc89b02) our documentation as I realized it may not be clear how to configure the local paths properly.

Let me know if you managed to get this working. If not, I am happy to get some feedback to make this easier for folks :)
